datasets/ziekte_plagen_exoten_groen/dataset.json

```
@@ -0,0 +1,303 @@
+{
+  "id": "ziektePlagenExotenGroen",
+  "type": "dataset",
+  "authorizationGrantor": "Directie V&OR, Gemeente Amsterdam",
+  "auth": "OPENBAAR",
+  "theme": [
+    "Duurzaamheid en milieu"
+  ],
+  "homepage": "https://data.amsterdam.nl",
+  "owner": "Directie V&OR, Gemeente Amsterdam",
+  "spatialDescription": "Gemeente Amsterdam",
+  "version": "1.0.0",
+  "title": "Ziekten, Plagen en Exoten Groen",
+  "language": "nl",
+  "dateCreated": "2023-01-01T00:00:00+01:00",
+  "license": "openbaar, tenzij anders aangegeven / behoudens uitzonderingen",
+  "creator": "Datateam Beeldschoon/BOR",
+  "publisher": "Datateam Beheer en Openbare Ruimte",
+  "description": "dataset met gegevens over ziekten, plagen en exoten in het groen in de gemeente Amsterdam",
+  "status": "beschikbaar",
+  "keywords": [
+    "eikenprocessierups",
+    "eikenboom",
+    "rups",
+    "groen"
+  ],
+  "crs": "EPSG:28992",
+  "temporalUnit": "",
+  "tables": [
+    {
+      "id": "eikenprocessierups",
+      "type": "table",
+      "auth": "OPENBAAR",
+      "license": "openbaar, tenzij anders aangegeven / behoudens uitzonderingen",
+      "title": "Eikenprocessierups",
+      "description": "Meldingen van de eikenprocessierups en informatie over de bestrijding hiervan",
+      "version": "1.0.0",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "mainGeometry": "geometrie",
+        "required": [
+          "schema",
+          "id"
+        ],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "geometrie": {
+            "$ref": "https://geojson.org/schema/Point.json",
+            "description": "Aanduiding voor de geometrie van het beheerobject volgens het Stelsel van de Rijksdriehoeksmeting (RD, epsg:28992). bevat punt-geometrie."
+          },
+          "boomid": {
+            "description": "Uniek identificatienummer van een boom",
+            "type": "string"
+          },
+          "gbdBuurtId": {
+            "type": "string",
+            "relation": "gebieden:buurten",
+            "uri": "https://www.amsterdam.nl/stelselpedia/gebieden-index/catalogus/buurt",
+            "description": "Unieke identificatie van een buurt"
+          },
+          "gbdBuurtCode": {
+            "type": "string",
+            "uri": "https://www.amsterdam.nl/stelselpedia/gebieden-index/catalogus/buurt",
+            "description": "Unieke code (hierin zie je de Stadsdeel- en Wijkcode terug)"
+          },
```

`gbdBuurtCode` is toch redundant als er al een relatie naar `gebieden:buurten` is? En `Id` aan het einde van een relatienaam zetten is niet nodig.

```suggestion
          "gbdBuurt": {
            "type": "string",
            "relation": "gebieden:buurten",
            "uri": "https://www.amsterdam.nl/stelselpedia/gebieden-index/catalogus/buurt",
            "description": "Unieke identificatie van een buurt"
          },
```

---

gbBuurtCode is idd een beetje dubbelop, maar heel handig voor de bi-ontwikkelaars die niet met de api werken maar op de database inprikken en/of met csv aan de slag gaan (zij joinen bv. hun eigen shapefiles/referentietabellen met buurt/wijk/gebied gegevens).

gbdBuurtId -> gbdBuurt is een goed idee. zal in mijn team communiceren dat het zo heurt.

---

Maar als ze toch een join gaan doen, dan kunnen ze toch meteen met de buurtentabel? Qua ruimtegebruik maakt het weinig uit; ik probeer te voorkomen dat hier inconsistenties ontstaan als de ene dataset al gewijzigd is maar de andere nog niet.

---

O, ik zie dat ie toch verwijderd is. Sorry!